pennylane/queuing.py

```diff
@@ -459,6 +459,20 @@ def circuit(x):
     >>> print(qml.draw(circuit)(0.6))
     0: ──RX(0.4)──RY(0.6)──RX(0.4)──┤ ⟨Z⟩
 
+    Note: if you use ``apply`` on an operator that has already been queued, it will
+    be queued for a second time. For example:
```

Comment:
```suggestion
    .. note ::
        If you use ``apply`` on an operator that has already been queued, it will
        be queued for a second time. For example:
```

Reply:
i considered a `warning` directive, but I thought the `code-block` directive would be ruined. tried locally and it looked fine, so I think I'll use it!

Reply:
It looks good for me locally as well. I definitely like this version the most. 👍